.circleci/config.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           paths: .
   publish:
     docker:
-      - image: circleci/node:12
+      - image: cimg/node:12
```

Comment:
Turns out that this broke our publishing pipeline. Could you fix it @anthonymayer 
I believe it should be `12.22`:

https://circleci.com/developer/images/image/cimg/node#image-tags

Reply:
@cortinico sorry about that, I thought they supported all the same tags. I'll fix it. I do wonder then if this needs to be considered a breaking change, since https://github.com/react-native-community/react-native-circleci-orb/pull/146/files#diff-0552beaefc6ae33d95cf39dcb9934ca462458a76f414573ac2b60c25926a8608R11 can take in versions which may no longer be compatible

Reply:
fix is here https://github.com/react-native-community/react-native-circleci-orb/pull/147

Reply:
> since [#146 (files)](https://github.com/react-native-community/react-native-circleci-orb/pull/146/files#diff-0552beaefc6ae33d95cf39dcb9934ca462458a76f414573ac2b60c25926a8608R11) can take in versions which may no longer be compatible

I think it might be the case yes. It's definitely worth noting somewhere in the release notes.